apps/ledger-live-desktop/src/renderer/families/polkadot/AccountHeaderManageActions.js

```diff
@@ -65,7 +65,7 @@ const AccountHeaderManageActions = ({ account }: Props) => {
 
   return [
     {
-      key: "polkadot",
+      key: "Stake",
```

Comment:
just out of interest, what's the reason for this change?

Reply:
never mind - i can see this is added as we use this to find the right stakeAction.onClick to call